torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1836,28 +1836,32 @@ Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const T
   auto gsigma = grads[1];
 
   auto u = raw_u;
-  auto v = raw_v;
+  // TODO: remove .conj()
+  // once https://github.com/pytorch/pytorch/issues/45821 is resolved
+  auto v = raw_v.conj();
   auto gu = grads[0];
-  auto gv = grads[2];
+  auto gv = grads[2].conj();  // TODO: remove .conj()
 
   if (!some) {
     // We ignore the free subspace here because possible base vectors cancel
     // each other, e.g., both -v and +v are valid base for a dimension.
     // Don't assume behavior of any particular implementation of svd.
     u = raw_u.narrow(-1, 0, k);
-    v = raw_v.narrow(-1, 0, k);
+    v = raw_v.narrow(-1, 0, k).conj();  // TODO: remove .conj()
     if (gu.defined()) {
       gu = gu.narrow(-1, 0, k);
     }
     if (gv.defined()) {
       gv = gv.narrow(-1, 0, k);
     }
   }
-  auto vt = v.transpose(-2, -1);
+  auto vh = v.conj().transpose(-2, -1);
```

Comment:
nit: you conjugate v above to conjugate it again here.
While I see that it matches the formulas, it looks a bit weird in this context.

Another option might be to name the input argument `conj_v` and work with that directly?

Reply:
I agree that it looks weird, I wanted to keep those extra `.conj()` changes only at the beginning of the function with the hope it'll be fixed soon and keep everything else matching the formulas and so that it is less confusing.